.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -384,8 +384,10 @@ jobs:
         - name: (Artifact re-build) Build Webpack
           if: steps.download_artifact_1.outcome == 'failure'
           run: python -m scripts.build --prod_env
-        - name: Run Acceptance Test ${{ matrix.suite }}
+        - name: Run Desktop Acceptance Test ${{ matrix.suite }}
           run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_acceptance_tests --skip-build --suite=${{ matrix.suite }} --prod_env
+        - name: Run Mobile Acceptance Test ${{ matrix.suite }}
+          run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_acceptance_tests --skip-build --suite=${{ matrix.suite }} --prod_env --mobile
```

Comment:
Should the screen be different here for mobile tests?

Reply:
No, the screen size can be bigger than what we are testing when we open the environment since puppeteer is the one setting the viewport on chrome.